.github/workflows/ci.yml

```
@@ -322,6 +322,47 @@ jobs:
         with:
           job-id: jdk-integration-tests-${{ matrix.os }}-${{ matrix.dist }}-${{ matrix.version }}
 
+  login-v1-tests:
+    name: Login Theme v1 tests
+    needs: build
+    timeout-minutes: 100
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        dist: [temurin]
+        version: [17]
+      fail-fast: false
+    runs-on: ${{ matrix.os }}
```

No need to define a matrix for a single combo

```suggestion
    runs-on: ubuntu-latest
```